webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getURIsearch } from 'foremanReact/common/urlHelpers';
+import { Spinner } from 'patternfly-react';
+import PageLayout from 'foremanReact/pages/common/PageLayout/PageLayout';
+import { getURIQuery } from 'foremanReact/common/helpers';
+import ExportButton from 'foremanReact/pages/common/PageLayout/components/ExportButton/ExportButton';
+import { STATUS } from 'foremanReact/constants';
+import TasksDashboard from '../TasksDashboard';
+import TasksTable from './TasksTable';
+import { resolveSearchQuery, addSearchToURL } from './TasksTableHelpers';
+import { TASKS_SEARCH_PROPS } from './TasksTableConstants';
+import './TasksTablePage.scss';
+
+const TasksTablePage = ({ ...props }) => {
```

Comment:
Does it make sense to do destruct the `getTableItems` from the `props` so the `TasksTable` won't receive it?
```js
const TasksTablePage = ({ getBreadcrumbs, ...props }) => { ... };
```

Reply:
Yup, changed!